install.sh

```diff
@@ -17,8 +17,16 @@ usage()
   echo "  --help            Print this help text."
 }
 
-# Default conda location
-conda_prefix=$HOME/miniforge3
+# {{{ Default conda location
+
+EXECUTABLE="$(readlink -f "$0")"
```

Comment:
readlink doesnt work in macos, you have to do some bash gymnastics (e.g. https://stackoverflow.com/questions/39340169/dir-cd-dirname-bash-source0-pwd-how-does-that-work?noredirect=1&lq=1)

Reply:
Thanks! 0426412